@xen-orchestra/fs/src/utils.js

```diff
@@ -0,0 +1,45 @@
+import { createLogger } from '@xen-orchestra/log'
+import { TimeoutError } from 'promise-toolbox'
+const { info, warn } = createLogger('xo:fs:abstract')
```

Comment:
I would add a blank line to separate imports from the rest of the code

```suggestion
import { TimeoutError } from 'promise-toolbox'

const { info, warn } = createLogger('xo:fs:abstract')
```